src/mon/FSCommands.cc

```diff
@@ -385,16 +385,16 @@ class SetHandler : public FileSystemCommandHandler
       return -EINVAL;
     }
 
-  bool confirm = false;
-  cmd_getval(cmdmap, "yes_i_really_mean_it", confirm);
-  if (var == "max_mds" && !confirm && mon->mdsmon()->has_any_health_warning()) {
-    ss << "One or more file system health warnings are present. Modifying "
-       << "the file system setting variable \"max_mds\" may not help "
-       << "troubleshoot or recover from these warnings and may further "
-       << "destabilize the system. If you really wish to proceed, run "
-       << "again with --yes-i-really-mean-it";
-    return -EPERM;
-  }
+    bool confirm = false;
+    cmd_getval(cmdmap, "yes_i_really_mean_it", confirm);
+    if (var == "max_mds" && !confirm && mon->mdsmon()->has_any_health_warning()) {
+      ss << "One or more file system health warnings are present. Modifying "
+	 << "the file system setting variable \"max_mds\" may not help "
+	 << "troubleshoot or recover from these warnings and may further "
+	 << "destabilize the system. If you really wish to proceed, run "
+	 << "again with --yes-i-really-mean-it";
+      return -EPERM;
+    }
```

Comment:
@batrick @vshankar You had approved the PR where this code block was added, see - https://github.com/ceph/ceph/pull/59420/files#diff-42ceb20fdeb082bdba8efcfa88a042d19932504d90a8c13c8cc0392daf9e12b3R388-R397